File: CodeWars/JavaScript/doublechar.js

//cara yang jauh lebih sederhana
function double(str) {
    return str.split("").map(i => String(i).repeat(2)).join("")
  }
  
  console.log(double("Baju")) // "BBaajjuu"

//cara awal
function doubleChar(str){

    //melakukan inisiasi hasil
    let hasil =""

    //melakukan iterasi agar char nya double 
    for (i=0;i<str.length;i++){
            hasil += str[i]+ str[i] 
    }
    console.log(hasil)
}

doubleChar("Baju")

File: CodeWars/Golang/doublechar.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	fmt.Println(double("Baju")) // BBaajjuu
}

func double(str string) string {
	var res []string
	strs := strings.Split(str, "")
	for _, v := range strs {
		res = append(res, strings.Repeat(v, 2))
	}
	return strings.Join(res, "")
}
